File: Java/Assignments/Assign_2/prog4.java
class Prog4 {

	public static void main(String[] args) {
		
		int arr[] = {10,8,56,23,45,97};
		int n = arr.length;
		int first = -1000, second = -1000;
		for (int i = 0; i < n; i++) {
			System.out.print(arr[i] + " ");	
		}
		System.out.println();
		if(n < 2)
			System.out.println("Invalid!!");

		for (int i = 0; i < n; i++) {
			if(arr[i] > first){
				second = first;
				first = arr[i];
			}				
			else if (arr[i] > second && arr[i] != first) {
				second = arr[i];
			}
		}
		System.out.println("Second max = " + second);
	}
}

File: Python/26August/prog4.py
print("Enter 3 numbers:",end=" ")
arr = [int(i) for i in input().split()]

first = arr[0] 
second = -10000 
third = -10000 
for i in range(1, len(arr)):  
    if (arr[i] > first): 
        third = second 
        second = first 
        first = arr[i] 
    elif (arr[i] > second): 
        third = second 
        second = arr[i] 
    elif (arr[i] > third): 
        third = arr[i] 
  
print("The third largest element is:", third) 